### PR TITLE
OSOE-78: UI Kit Showcase UI test

### LINKED ISSUE #8: Add a UI test for the Showcase (OSOE-78)
Add a UI test extension method and a corresponding test in OSOCE that opens the Showcase page and plays around with it a bit.

### PR DESCRIPTION
**Pull request by Luko6**
OSOE-78

"Fixes https://github.com/Lombiq/Orchard-UIKit/issues/8"

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,7 +4,8 @@
 obj/
 bin/
 artifacts/
-wwwroot/
+./Lombiq.UIKit/wwwroot/
```
Comment (wAsnk) [inline .gitignore]: Use `**/wwwroot/*` instead.

**Review by wAsnk — CHANGES_REQUESTED** [review]
I got this while trying to build the solution on a clean repo
image
Could you try building it with a clean repo?

**Comment by wAsnk** [comment]
> I got this while trying to build the solution on a clean repo image Could you try building it with a clean repo?

This doesn't seems to be good:
image

File: .gitignore (hunk 1 above)
Comment (0liver) [reply .gitignore]: Why is a change here needed anyway?

Comment (Luko6) [reply .gitignore]: @wAsnk yes.
@0liver the project was made into a solution with multiple projects, like others in OSOCE, please check it if interested.

**Comment by Luko6** [comment]
Builds for me and CI, so the problem must be locally at you, you need the `OSOCE-78-re` branch not `OSOCE-78` that may be a problem.

**Comment by wAsnk** [comment]
> Builds for me and CI, so the problem must be locally at you, you need the `OSOCE-78-re` branch not `OSOCE-78` that may be a problem.

It's not working here as there are no such modules that you are referencing. It's another thing in the OSOCE repo.

**Comment by wAsnk** [comment]
You have conflicts here and in the OSOCE PR.

**Comment by Luko6** [comment]
Ok I understand your comment now. If you try to build the `dev` branch here, the same problem comes up. If you check [Chart.js](https://github.com/Lombiq/Orchard-Chart.js) which has a similar structure to this module, the same happens again. This is not meant for standalone builds, and OSOCE is an example on how this should be used, this is even documented in the Readme.

**Comment by wAsnk** [comment]
> Ok I understand your comment now. If you try to build the `dev` branch here, the same problem comes up. If you check [Chart.js](https://github.com/Lombiq/Orchard-Chart.js) which has a similar structure to this module, the same happens again. This is not meant for standalone builds, and OSOCE is an example on how this should be used, this is even documented in the Readme.

This is soooo ugly. 😅 But I accept it. 
Could you just add to the readme where those dependencies should be placed exactly if not the NuGet versions are used?

**Comment by wAsnk** [comment]
I get this error when I run the default task in the Lombiq.UIKit:
image
Am I doing something wrong again? :D

**Comment by wAsnk** [comment]
When I'm using the bootstrap dropdown buttons, the dropdown can't be closed:
image

File: Lombiq.UIKit.Tests.UI/License.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+Copyright © 2021, [Lombiq Technologies Ltd.](https://lombiq.com)
```
Comment (Piedone) [inline Lombiq.UIKit.Tests.UI/License.md:1]: Use the updated license files from the root in all cases.

**Comment by Piedone** [comment]
Add "Fixes #8" to the PR description to automatically close the related issue.

**Review by wAsnk — APPROVED** [review]

**Pull request merged**